fclient/federationclient.go

```diff
@@ -736,3 +738,31 @@ func (ac *federationClient) RoomHierarchy(
 	}
 	return
 }
+
```

Comment:
I think documentation in the same spirit as the other methods would be appropriate.

Reply:
Fair, and good to document that the caller of this method is responsible to close the returned response body, if any.